### DISCUSSION TITLE
Antonina Ivanova?

### DISCUSSION THREAD

**Discussion by Doug777-colab** [p0deje/Maccy#990] [Q&A] [answered]
Hi, I am a long term Maccy user and you certainly have my trust. I am looking to support and also gain the convenience  of Mac App Store (MAS) updates. However, I notice the developer on MAS is Antonina Ivanova, not you. Antonina is not connected with any other MAS apps. 

Can you please confirm:

- If Antonina is your offical distribution of Maccy on MAS?
- If a third party is involved in distributing Maccy on MAS for you (Antonina, whether they are a developer or development team), what controls and oversight do you have to prevent changes to your intended distribution?

I am thinking of this from a security perspective where if there is some risk, I might be best to continue with Maccy direct from the source/you :-)

Thank you

Doug

**Comment by p0deje** [comment, marked as answer]
It's an official distribution team and any changes that are made for the App Store version are accessible via https://github.com/p0deje/Maccy/tree/appstore branch. There are no other changes that could be done by third parties. There is absolutely no risk in installing the App Store version.
